cli/tests/pcluster_config/test_pcluster_config_convert/test_slurm_sit_simple/expected_output.ini

```diff
@@ -3,7 +3,6 @@ vpc_id = vpc-6b7c3810
 master_subnet_id = subnet-1956ef16
 
 [aws]
```

Comment:
Let's remove the [aws] section from the source and the expected output

Reply:
This section is added by our code. You mean I should try to amend our code to stop generate this section if nothing is inside?

Reply:
I think it's added because it's in the initial file. We can remove it from there and be sure that aws section is not written if there are no values.

Reply:
Since now we are raising errors if no region is found, the expected_output has to have a region specified. Do we want to handle ``pcluster_config`` differently, i.e. allow running ``pcluster_config`` even if there is no region found?